libraries/standard/mqtt/cbmc/proofs/Makefile-project-defines

```diff
@@ -0,0 +1,32 @@
+# -*- mode: makefile -*-
+# The first line sets the emacs major mode to Makefile
+
+################################################################
+# Use this file to give project-specific definitions of the command
+# line arguments to pass to CBMC tools like goto-cc to build the goto
+# binaries and cbmc to do the property and coverage checking.
+#
+# Use this file to override most default definitions of variables in
+# Makefile.common.
+################################################################
+
+# Flags to pass to goto-cc for compilation (typically those passed to gcc -c)
+COMPILE_FLAGS += \
+        -fPIC \
+        -std=gnu90
```

Comment:
Please consider using the += style.  I saw some evidence with HTTP that the line continuation style does not interact with the new Makefile.common macros that my team has developed.

```
COMPILE_FLAGS += -fPIC 
COMPILE_FLAGS += -std=gnu90

INCLUDES += -I$(SRCDIR)/libraries/standard/mqtt/cbmc/include 
INCLUDES += -I$(SRCDIR)/libraries/standard/mqtt/include 
INCLUDES += -I$(SRCDIR)/libraries/standard/utilities/include 
INCLUDES += -I$(SRCDIR)/libraries/standard/mqtt/src 
INCLUDES += -I$(SRCDIR)/demos/logging-stack 
INCLUDES += -I$(SRCDIR)/demos/mqtt/mqtt_demo_plaintext

DEFINES += -Dmqtt_EXPORTS
```

Reply:
OK